config/locales/en.yml

```
@@ -2463,6 +2463,7 @@ en:
   error_invalid_selected_value: "Invalid selected value."
   error_journal_attribute_not_present: "Journal does not contain attribute %{attribute}."
   error_pdf_export_too_many_columns: "Too many columns selected for the PDF export. Please reduce the number of columns."
+  error_pdf_date_range_too_long: "The selected work package date range exceeds the allowable limit for PDF export. Please condense the range to a maximum of %{years} years."
```

That sentence sounds slightly off. I am under the impression that there's either an article missing in front of "PDF export" -> "the allowable limit for a PDF export." or that we should use plural here -> "the allowable limit for PDF exports."

Another option would be to flip the sentence around to avoid this issue altogether:

```suggestion
  error_pdf_date_range_too_long: "The selected work package date range exceeds the allowable PDF export limit. Please condense the range to a maximum of %{years} years."
```

I might be wrong and the original sentence is completely valid. You be the judge of that ^^

---

Thanks for the suggestion. I agree, your version sounds more straightforward. I commit it and merge. 👍